lib/cloud_controller/diego/ssh_key.rb

```diff
@@ -21,7 +22,9 @@ def authorized_key
           end
         end
 
-        delegate :fingerprint, to: :key
+        def fingerprint
+          ::SSHKey.new(key.to_der).sha1_fingerprint
```

Comment:
Do we know if this is backward compatible with earlier clients?

Reply:
Who is the client in this case? It was my understanding that these fingerprints get validated by the ssh-proxy but do not reach the cf cli.

Reply:
SHA1 support in the ssh-proxy was added 8 years ago: https://github.com/cloudfoundry/diego-ssh/commit/0f6f64f1d6db9e3c1888c98196aefab7da7a9888#diff-1d74870266949f1927a500e5f0002617239d3e3cd9bcceee6be014738e5e0f2bR230

Reply:
Thanks for providing context.